go/ql/lib/ext/fmt.model.yml

```
@@ -8,3 +8,14 @@ extensions:
       - ["fmt", "ScanState", True, "Token", "", "", "Argument[-1]", "ReturnValue[0]", "taint", "manual"]
       - ["fmt", "State", True, "Write", "", "", "Argument[0]", "Argument[-1]", "taint", "manual"]
       - ["fmt", "Stringer", True, "String", "", "", "Argument[-1]", "ReturnValue", "taint", "manual"]
+      - ["fmt", "", True, "Append", "", "", "Argument[0].ArrayElement", "ReturnValue", "taint", "manual"]
```

Even though the signature is `func Append(b []byte, a ...any) []byte`) I think we just want this:
```suggestion
      - ["fmt", "", True, "Append", "", "", "Argument[0]", "ReturnValue", "taint", "manual"]
```
And the same for the other `Append` models

---

I guess we don't expect individual bytes of the `[]byte` parameter to be tainted, right?